lighthouse-core/test/gather/driver-test.js

```diff
@@ -46,9 +48,23 @@ function createActiveWorker(id, url, controlledClients, status = 'activated') {
   };
 }
 
+function createOnceMethodResponse(method, response) {
+  assert.deepEqual(!!sendCommandMockResponses[method], false, 'stub response already defined');
```

Comment:
`deepEqual` left over from an earlier revision?

Reply:
I wrote the method assuming someone else may try to use it. And just wanted to assert no one was going to use it twice.
I can remove though if you don't see the value.

Reply:
> I can remove though if you don't see the value.

sorry, I think it's fine, I meant the `deepEqual` is kind of overkill because they're both booleans :)